library/pom.xml

```diff
@@ -372,9 +375,37 @@
 							</execution>
 						</executions>
 					</plugin>
+
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<!-- These plugins can't be part of the standard build -->
+			<id>ckeditor-extras</id>
+			<activation>
+				<activeByDefault>false</activeByDefault>
+				<property>
+					<name>ckeditor-extras</name>
+				</property>
+			</activation>
+			<properties>
+				<ckeditor-extra-plugins>,a11ychecker</ckeditor-extra-plugins>
```

Comment:
Yeah I think we should have one property to enable them all, but I think the property for each plugin (After it gets enabled) should be more unique like you have here
<ckeditor-extra-plugins>,a11ychecker</ckeditor-extra-plugins>

This is a property specific to adding in the a11ychecker.